resources-domain/resources-service/src/import/importService.ts

```diff
@@ -132,7 +132,7 @@ const importService = () => {
           resources,
         );
         if (error.validationFailure) {
-          return { ...error.validationFailure, resource: error.resource };
+          return { ...error.validationFailure, resource: JSON.stringify(error.resource) };
```

Comment:
A non pretty printed JSON string should just get truncated on one line rather than spread across many logs